composer.json

```diff
@@ -26,8 +26,8 @@
     "prefer-stable": true,
     "require": {
         "php": ">=7.2.0",
-        "illuminate/contracts": "^6.0",
-        "illuminate/support": "^6.0"
+        "illuminate/contracts": "^6.0|5.8.*",
```

Comment:
Could you switch these around, to read `5.8.*|^6.0`. Same for `illuminate/support`.